service/controller/legacy/v29/ebs/error.go

```diff
@@ -5,6 +5,10 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
+var executionFailedError = &microerror.Error{
```

Comment:
Let's keep that in sync like in https://github.com/giantswarm/aws-operator/pull/1796.

```suggestion
// executionFailedError is an error type for situations where Resource execution
// cannot continue and must always fall back to operatorkit.
//
// This error should never be matched against and therefore there is no matcher
// implement. For further information see:
//
//     https://github.com/giantswarm/fmt/blob/master/go/errors.md#matching-errors
//
var executionFailedError = &microerror.Error{
```